src/date-picker/calendar.jsx

```diff
@@ -92,6 +92,7 @@ const Calendar = React.createClass({
     let styles = {
       root: {
         fontSize: 12,
+        userSelect: 'none',
```

Comment:
I think that you can stay with the `cursor: default` here instead of spreading it around.
Would this work?